static/erdblick/view.js

```diff
@@ -0,0 +1,252 @@
+"use strict";
+
+import {ErdblickModel, MapViewerViewport} from "./model.js";
+import {FeatureWrapper} from "./features.js";
+
+export class ErdblickView
```

Comment:
I remember there being some issue with the attribute panel, maybe that it disappeared unexpectedly when clicked on for navigation... is it fixed now?

Reply:
I think so 👀